### PR TITLE
Upgrade js-yaml to 4.X

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

closed #3479

### CODE REVIEW

File: src/common/k8s-api/endpoints/helm-releases.api.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export async function createRelease(payload: IReleaseCreatePayload): Promise<IRe
   const { repo, ...data } = payload;
 
   data.chart = `${repo}/${data.chart}`;
-  data.values = jsYaml.safeLoad(data.values);
+  data.values = yaml.load(data.values);
```
Comment (Nokel81) [inline src/common/k8s-api/endpoints/helm-releases.api.ts]: @jakolehm The typings for js-yaml@4 now have `yaml.load` return `string | number | object`. However, `data.values` is supposed to be a string, do you remember why the previous call was being made? 

Note: the migration guide basically says that the default should now be the safe version https://github.com/nodeca/js-yaml/blob/49baadd52af887d2991e2c39a6639baa56d6c71b/migrate_v3_to_v4.md

Comment (jakolehm) [reply src/common/k8s-api/endpoints/helm-releases.api.ts]: @nevalla ping

Comment (nevalla) [reply src/common/k8s-api/endpoints/helm-releases.api.ts]: I think it took values as string and converted it as object

**Review by nevalla — DISMISSED** [review]
Code it self looks good

**Review by nevalla — APPROVED** [review]

**Pull request merged**